components/org.wso2.identity.webhook.common.event.handler/src/main/java/org/wso2/identity/webhook/common/event/handler/LoginEventHookHandler.java

```diff
@@ -0,0 +1,189 @@
+        }
+        return false;
+    }
+
+    private boolean isSupportedEvent(String eventName) {
+
+        return IdentityEventConstants.EventName.AUTHENTICATION_SUCCESS.name().equals(eventName) ||
+                IdentityEventConstants.EventName.AUTHENTICATION_STEP_FAILURE.name().equals(eventName) ||
+                IdentityEventConstants.EventName.AUTHENTICATION_FAILURE.name().equals(eventName);
+    }
+
+    @Override
+    public void handleEvent(Event event) throws IdentityEventException {
+
+        EventData eventData = buildEventDataProvider(event);
+
+        if (eventData.getAuthenticationContext().isPassiveAuthenticate()) {
+            return;
+        }
+
+        //TODO: Add the implementation to read the Event Schema Type from the Tenant Configuration
+        LoginEventPayloadBuilder payloadBuilder = PayloadBuilderFactory
+                .getLoginEventPayloadBuilder(EVENT_SCHEMA_TYPE_WSO2);
+        EventAttribute loginEventPublisherConfig = getLoginEventPublisherConfigForTenant(
+                eventData.getAuthenticationContext().getLoginTenantDomain(), event.getEventName());
+        EventPayload eventPayload;
+        String eventUri;
+
+        if (IdentityEventConstants.EventName.AUTHENTICATION_SUCCESS.name().equals(event.getEventName()) &&
+                loginEventPublisherConfig.isPublishEnabled()) {
+            eventPayload = payloadBuilder.buildAuthenticationSuccessEvent(eventData);
+            eventUri = EventHookHandlerUtils.getEventUri(Constants.EventHandlerKey.LOGIN_SUCCESS_EVENT);
+            String tenantDomain = eventData.getAuthenticationContext().getLoginTenantDomain();
+            SecurityEventTokenPayload securityEventTokenPayload = buildSecurityEventToken(eventPayload,
+                    eventData.getAuthenticationContext(), eventUri);
+            EventHookHandlerUtils.publishEventPayload(securityEventTokenPayload, tenantDomain, eventUri);
+        } else if (IdentityEventConstants.EventName.AUTHENTICATION_STEP_FAILURE.name().equals(event.getEventName()) &&
+                loginEventPublisherConfig.isPublishEnabled()) {
+            eventPayload = payloadBuilder.buildAuthenticationFailedEvent(eventData);
+            eventUri = EventHookHandlerUtils.getEventUri(Constants.EventHandlerKey.LOGIN_FAILED_EVENT);
+            String tenantDomain = eventData.getAuthenticationContext().getLoginTenantDomain();
+            SecurityEventTokenPayload securityEventTokenPayload = buildSecurityEventToken(eventPayload,
+                    eventData.getAuthenticationContext(), eventUri);
+            EventHookHandlerUtils.publishEventPayload(securityEventTokenPayload, tenantDomain, eventUri);
+        }
+    }
+
+    /**
+     * Check whether the login event handler is enabled.
+     *
+     * @return True if the login event handler is enabled.
+     */
+    public boolean isLoginEventHandlerEnabled() {
+
+        String enablePropertyKey = Constants.LOGIN_EVENT_HOOK_NAME + "." + Constants.ENABLE;
```

Comment:
enablePropertyKey seems to be a string constant

Reply:
Addressed with https://github.com/wso2-extensions/identity-webhook-event-handlers/pull/1/commits/6e511b91f52c4b71ccff73c989894c76b285ebf0